src/fe.c

```diff
@@ -26,7 +26,11 @@ static uint64_t load_4(const unsigned char *in) {
     return result;
 }
 
-
+static int64_t shift_left(int64_t v, int s) {
```

Comment:
this function is expected to disappear under optimizations, and hence should have internal linkage